extension/content.js

```diff
@@ -51,6 +52,24 @@ function addReleasesTab() {
 	}
 }
 
+function addBlameParentLinks() {
+	let pathname = location.pathname;
+
+	$('.blame-sha').each(function(index, commitLink) {
+		let $commitLink = $(commitLink)
+		let $blameParentLink = $commitLink.clone();
+		let commitSha = commitLink.href.match(/(?<=commit\/)\w+/)[0];
```

Comment:
It's just the nature of manipulating a DOM you don't own. When doing DOM manipulation/targeting specificity, accuracy, and fragility are directly related. I suggest being specific so it works and then deal with bugs from the DOM changing (if they happen) later. My 2¢ anyway, but half this plugin would break if the GH markup changes anyway.